frontend/src/pages/join/[shortname].tsx

```diff
@@ -122,25 +122,32 @@ function Join({ shortname }): React.ReactElement {
 
   const onToggleSidebarVisible = (): void => setSidebarVisible((prev): boolean => !prev)
 
-  // handle creation of a new confusion timestep
-  // const onNewConfusionTS = _debounce(
-  //   async ({ difficulty = 0, speed = 0 }): Promise<void> => {
-  //     try {
-  //       newConfusionTS({
-  //         variables: {
-  //           difficulty,
-  //           fp: fingerprint,
-  //           sessionId,
-  //           speed,
-  //         },
-  //       })
-  //     } catch ({ message }) {
-  //       console.error(message)
-  //     }
-  //   },
-  //   4000,
-  //   { trailing: true }
-  // )
+  // handle creation of a new confusion timestep with debounce for aggregation
+  const onNewConfusionTS = _debounce(
+    async ({ difficulty = 0, speed = 0 }): Promise<void> => {
+      try {
+        newConfusionTS({
+          variables: {
+            speed,
+            difficulty,
+            fp: fingerprint,
+            sessionId,
+          },
+        })
+        window.sessionStorage?.setItem(
```

Comment:
tried that before, but there were some issues with data that was overwritten in a strange fashion as one would have to use state for the other of the two variables. I could give it another try, but this will required handling of more different cases - e.g. only one of the attributes was set before the reload, etc.
Only option to go around these issues: specify two separate storage items for the two values

Reply:
also change to localForage